dappnode/DNP_DAPPMANAGER
# Remove openethereum client

## Context

Openethereum will be deprecated, it will not be available in the dappstore and users must start migrating to other clients

## Approach

Startup migration to deprecate openethereum. It will remove the package with all the volumes and if the repository selected was openethereum, change the `ethClientTarget` based on the following preference: geth/light-geth > nethermind > remote
Remove all migration of ethchain. Ethchain is not longer in the dappstore and last publishe was 2 years ago

## Test instructions

- Install open Ethereum and select the repository client openethereum. Then update to a dappmanager with this code. Make sure the open Ethereum was removed with the volumes and the client selected in the repository was set to remote
- Install openEthereum and  geth full node. Then update to a dappmanager with this code. Make sure the open Ethereum was removed with the volumes and the client selected in the repository was set to geth full node
- Install openEthereum and  geth light client. Then update to a dappmanager with this code. Make sure the open Ethereum was removed with the volumes and the client selected in the repository was set to geth light client
- Install openEthereum and  nethermind. Then update to a dappmanager with this code. Make sure the open Ethereum was removed with the volumes and the client selected in the repository was set to nethermind
